fixtures/beamz/ps10w-led-pin-spot-10w-4-in-1-dmx.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "PS10W LED PIN SPOT 10W 4-IN-1 DMX",
+  "shortName": "PS10W",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Raine M. Ekman"],
+    "createDate": "2024-05-30",
+    "lastModifyDate": "2024-05-30"
+  },
+  "links": {
+    "manual": [
+      "https://www.beamzlighting.com/wp-content/uploads/2024/05/151.259-PS10W-LED-Pin-Spot-10W-RGBW-DMX_manual_V1.3.pdf"
+    ],
+    "productPage": [
+      "https://www.beamzlighting.com/product/ps10w-led-pin-spot-10w-4-in-1-dmx/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=Mb6_FTCzp1o"
+    ]
+  },
+  "physical": {
+    "dimensions": [68, 145, 150],
+    "weight": 0.7,
+    "power": 17,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "capability": {
```

Comment:
For the color channels, the manual says 0=NoFun, and 1=0%, but I agree that an ordinary ColorIntensity channel is the way to go.